src/ensembl/src/content/app/entity-viewer/shared/helpers/transcripts-sorter.ts

```diff
@@ -40,11 +40,13 @@ function compareTranscriptLengths(
 
 export function defaultSort(transcripts: Transcript[]) {
   const proteinCodingTranscripts = transcripts
-    .filter((transcript) => transcript.biotype === 'protein_coding')
+    .filter((transcript) => getBiotype(transcript) === 'protein_coding')
     .sort(compareTranscriptLengths);
 
   const nonProteinCodingTranscripts = sortBy(
-    transcripts.filter((transcript) => transcript.biotype !== 'protein_coding'),
+    transcripts.filter(
+      (transcript) => getBiotype(transcript) !== 'protein_coding'
+    ),
     ['biotype']
   );
```

Comment:
Is this the right sorting algorithm for non-coding transcripts? Sorting them by the biotype field means that they just get sorted alphabetically. Is this what's expected?

Reply:
It should be sorted according to this ticket: [ENSWBSITES-537](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-537)

Reply:
For now yes, we sort the non-coding transcripts alphabetically. This could change in the future.